packages/documents/graphql/resolvers/_queries/documents.js

```diff
@@ -21,15 +21,35 @@ module.exports = async (_, args, { user, redis }) => {
     : 'publication'
 
   const repoIds = await redis.smembersAsync('repos:ids')
+  const {
+    feed
+  } = args
 
   return Promise.all(
     repoIds.map(repoId => {
       return redis.getAsync(`repos:${repoId}/${ref}`)
     })
   )
-    .then(publications => publications
-      .filter(Boolean)
-      .map(publication => JSON.parse(publication).doc)
-      .sort((a, b) => descending(new Date(a.meta.publishDate), new Date(b.meta.publishDate)))
+    .then(publications => {
+      let documents = publications
+        .filter(Boolean)
+        .map(publication => JSON.parse(publication).doc)
+      if (feed) {
+        documents = documents.filter(d => (
+          d.meta.feed ||
+          (d.meta.feed === undefined && d.meta.template === 'article')
+        ))
+      }
+      if (feed) {
+        documents = documents.filter(d => (
+          d.meta.feed ||
+          (d.meta.feed === undefined && d.meta.template === 'article')
+        ))
+      }
```

Comment:
remove duplicate

Reply:
forced pushed an update with the de-duped code.